parse_lib.py

```diff
@@ -95,8 +94,9 @@ def resolve_relative_resource_urls(html_string):
     imgs = html.find_all("img", src=True)
     equations = html.find_all("object", data=True)
     list(map(update_anchor_href, anchors))
+    equations_as_imgs = list(map(partial(convert_to_img, html), equations))
```

Comment:
I think we should add a comment explaining why we are converting `<object>`s to `<img>`s.